src/test/proxy/worker_proxy.py

```diff
@@ -0,0 +1,253 @@
+from typing import List
+from unittest import TestCase
+
+from ja.common.job import Job, JobStatus, JobPriority, JobSchedulingConstraints
+from ja.common.docker_context import DockerContext, DockerConstraints
+from ja.common.message.worker import WorkerResponse
+from ja.server.proxy.proxy import IWorkerProxy
+from test.abstract import skipIfAbstract
+
+
+class WorkerProxyDummy(IWorkerProxy):
+    def __init__(self, uid: str, jobs: List[Job]):
+        self._uid = uid
+        self._jobs: List[Job] = []
+        for job in jobs:
+            job_copy = Job.from_dict(job.to_dict())
```

Comment:
Why not use deepcopy?

Reply:
This isn't just to create a copy but also to simulate the transformation of object -> YAML and YAML -> object that will take place in the real proxy.

Reply:
makes sense